docusaurus.config.js

```diff
@@ -124,298 +125,321 @@ const injectLinkParseForEndpoint = ([, info]) => {
+                                <a target="_blank" href="https://github.com/apache/pulsar/"><img src="/img/github-white.svg" width="26"></a>
+                              </div>
+                              `,
+            },
+          ],
+        },
+      ],
+      copyright: `
+                            <div><img class="footer-apache-logo" src="/img/feather-logo-white.svg" alt="" width="20">Apache Foundation</div>
+                            <p>Apache Pulsar is available under the Apache License, version 2.0. Apache Pulsar is an open-source, distributed messaging and streaming platform built for the cloud.</p>
+                            <p>Copyright © ${new Date().getFullYear()} The Apache Software Foundation. All Rights Reserved. Apache, Pulsar, Apache Pulsar, and the Apache feather logo are trademarks or registered trademarks of The Apache Software Foundation.</p>
+                        `,
+    },
+    prism: {
+      theme: require("prism-react-renderer/themes/dracula"),
+      additionalLanguages: [
+        "csharp",
+        "groovy",
+        "http",
+        "ini",
+        "java",
+        "powershell",
+        "properties",
+        "protobuf",
+        "yaml",
       ],
+    },
+    algolia: {
+      appId: "WK2YL0SALL",
+      apiKey: "42d24d221fbd8eb59804a078208aaec0",
+      indexName: "apache_pulsar",
+    },
+  },
+  presets: [
+    [
+      "@docusaurus/preset-classic",
+      /** @type {import('@docusaurus/preset-classic').Options} */
+      {
+        docs: {
+          path: "docs",
+          sidebarPath: require.resolve("./sidebars.js"),
+          editUrl: `${githubSiteUrl}/edit/main`,
+          remarkPlugins: [
+            linkifyRegex(
+              /{\@inject\:\s?(((?!endpoint)[^}])+):([^}]+):([^}]+)}/,
+              injectLinkParse
+            ),
+            linkifyRegex(
+              /{\@inject\:\s?endpoint\|([^}]+)}/,
+              injectLinkParseForEndpoint
+            ),
+            math,
+          ],
+          rehypePlugins: [katex],
+          versions: versionsMap,
+          onlyIncludeVersions: buildVersions || ["current"],
+        },
+        blog: {
+          showReadingTime: true,
+          editUrl: `${githubSiteUrl}/edit/main/`,
+        },
+        theme: {
+          customCss: [
+            require.resolve("./src/css/custom.css"),
+            require.resolve("./src/css/docs.css"),
+            require.resolve("./src/css/base-table.css"),
+            require.resolve("./src/css/typography.css"),
+            require.resolve("./src/css/announcement-bar.css"),
+            require.resolve("./src/css/navbar.css"),
+            require.resolve("./src/css/footer.css"),
+            require.resolve("./src/css/variables.css"),
+          ],
+        },
+        googleAnalytics: {
+          trackingID: "UA-102219959-1",
+        },
+      },
+    ],
+  ],
+  plugins: [
+    [
+      "content-docs",
```

Comment:
It seems we are missing the `contribute` instance here:

```js
        [
            'content-docs',
            /** @type {import('@docusaurus/plugin-content-docs').Options} */
            ({
                id: 'contribute',
                path: 'contribute',
                routeBasePath: 'contribute',
                showLastUpdateAuthor: true,
                showLastUpdateTime: true,
                sidebarPath: require.resolve('./sidebarsDevelopment.js'),
                editUrl: `${githubSiteUrl}/edit/main`,
            }),
        ],
```

Reply:
Yes, thank you!
🤦‍♂️